doc update - darwin build

navidys opened the pull request:
updating README.md and install.md to include darwin build. 

Signed-off-by: Navid Yaghoobi <navidys@fedoraproject.org>

rhatdan commented on README.md:
```diff
@@ -21,7 +21,7 @@ It is using [podman go bindings](https://github.com/containers/podman/tree/main/
 ## Installation
 
-Building from source (Linux, Windows) or installing packaged versions are detailed in [install guide](install.md).
+Building from source (Linux, Windows and Darwin) or installing packaged versions are detailed in [install guide](install.md).
```
You sure you want to call this Darwin, or just Mac?

navidys replied:
You are right, I think it is better to call it Mac, will update the PR and change it to Mac.

navidys commented:
@rhatdan I have updated README.md and install.md and replaced "darwin" with "MacOS". 

Shall I update the Makefile and change "binary-darwin" target  to "binary-mac" ?

navidys commented on install.md:
```diff
@@ -2,36 +2,47 @@
 - [**Installing Packaged Versions**](#installing-packaged-versions)
-  - [**Arch Linux (AUR)**](#arch-linux-aur)
   - [**Fedora**](#fedora)
+  - [**Arch Linux (AUR)**](#arch-linux-aur)
```
My mistake, I was going to add two different sections:
1. offical repository (fedora, ..)
2. user repository (aur, copr and snap) 

will revert it back,  thanks for letting me know.

rhatdan commented:
LGTM
